src/components/Profile/myPosts/MyPosts.jsx

```diff
@@ -3,20 +3,30 @@ import s from "./MyPosts.module.css";
 import Post from "./Post/Post";
 
 const MyPosts = () => {
+  let postData = [
```

Comment:
```suggestion
  const postData = [
```
такі дані краще робити за компонентом